### ISSUE TITLE
Migrate stuff in readme into documentation

### ISSUE THREAD

**Issue by ghost** [O365/python-o365#844]
I suggest that we put most of the contents of the readme into the documentation, because the huge readme is hard to navigate and readme's aren't supposed to huge anyway. Most of the information should be in the documentation.

**Comment by janscas** [comment]
You are absolutely right, but I don’t have the time to do that. PR are very welcome tho

**Comment by alejcas** [comment]
Docs are updated and online
